spec/ace.md

```diff
@@ -33,6 +33,8 @@ An app's filesystem must be *rendered* in an empty directory by the following pr
 
 If during rootfs extraction a path is already present in the target directory from an earlier dependency, the previously extracted path MUST be overwritten. If the existing path is a symbolic link to a directory, the link MUST NOT be followed and it MUST be removed and replaced with the new path.
 
+If the optional field `readOnlyRootFS` is set to `true` in the ACI, the rootfs will be mounted as read-only.
```

Comment:
Nit, could you clarify line 26 where it says read write. Perhaps just add "by default"